modules/enet/networked_multiplayer_enet.cpp

```diff
@@ -74,8 +74,8 @@ int NetworkedMultiplayerENet::get_last_packet_channel() const {
 Error NetworkedMultiplayerENet::create_server(int p_port, int p_max_clients, int p_in_bandwidth, int p_out_bandwidth) {
 
 	ERR_FAIL_COND_V(active, ERR_ALREADY_IN_USE);
-	ERR_FAIL_COND_V(p_port < 0 || p_port > 65535, ERR_INVALID_PARAMETER);
-	ERR_FAIL_COND_V(p_max_clients < 0, ERR_INVALID_PARAMETER);
+	ERR_FAIL_COND_V(p_port < 1 || p_port > 65535, ERR_INVALID_PARAMETER);
```

Comment:
The docs still say "The port needs to be an available, unused port between 0 and 65535", which conflicts with this check.

Reply:
Right, thanks for spotting it.
Actually, I changed it back to allow port `0`.
That is a valid entry but with special meaning, i.e. asks the OS to assign a random available port.
For now, in godot, you can't really know what port you were bound to (no `getsockname` support for now), so I feel that needs to be added instead of disallowing port 0.